docs/proposals/organizations.md

```diff
@@ -0,0 +1,125 @@
+# Organizations
+
+## Problems
+ 1.  We need to be able to allocate quota to a group of projects.
+ 2.  We need to be able to limit a particular project's usage of shared quota (think htb rate/ceil for tc qdisc classes).
+ 3.  We need to be able to manage groups across related projects, but not across the entire cluster.
+ 4.  We need to be able to limit how many projects a given User/Organization can have.  (limits provisioning)
+
+We can do this with an Organization entity that can manage multiple projects.  An Organization can have multiple org-admins who are allowed to manage quota allocation to owned projects and OrgGroups (groups that scoped to projects owned by an Organization).  A project is owned by at most one Organization, an Organization can own muliple projects.
+
+### Open Questions
+ 1.  Should we have a two layer scoping `/api/v1/organization/<org name>/namespace/<namespace name>` or keep the current namespace structure?
+
+     Keeping the current namespace structure mitigates changes, but prevents two orgs from having the same namespace and makes storing additional org-scoped resources like quota and groups more difficult.
+     I'm in favor of keeping the current structure, but allowing deeper nesting of subresources and having named subresources.
+
+ 2.  Should we allow projects to be owned by Users or only allow ownership by Organizations?
```

Comment:
Downsides - a user has to have an organization first.  Which means we have to auto provision an organization.

Question: If we don't have projects owned by organizations, how does a user "upgrade" to be under an organization?